Src/WitsmlExplorer.Frontend/components/Sidebar/ServerManager.tsx

```diff
@@ -121,79 +134,126 @@ const ServerManager = (): React.ReactElement => {
-          value={selectedServer?.id ?? ""}
-          onOpen={() => {
-            setIsOpen(true);
-          }}
-          onClose={() => setIsOpen(false)}
-          MenuProps={{ open: isOpen }}
-        >
-          {servers
-            .sort((a, b) => a.name.localeCompare(b.name))
-            .map((server: Server) => (
-              <MenuItem value={server.id} key={server.id} onClick={() => onSelectItem(server)}>
-                <Typography style={{ marginRight: 20 * +isOpen, overflow: "hidden" }} color={"initial"}>
-                  {server.name}
-                </Typography>
-                {isOpen && (
-                  <ListItemSecondaryAction onClick={() => onEditItem(server)}>
-                    <Icon name="edit" color={colors.interactive.primaryResting} />
-                  </ListItemSecondaryAction>
-                )}
-              </MenuItem>
-            ))}
-          <Divider />
-          <MenuItem value={NEW_SERVER_ID} key={NEW_SERVER_ID} onClick={() => onEditItem(emptyServer())}>
-            <ListItemIcon>
-              <Icon name="add" color={colors.interactive.primaryResting} />
-            </ListItemIcon>
-            <Typography color={"initial"}>Add server</Typography>
-          </MenuItem>
-        </Select>
-
-        <AuthenticationState />
-      </FormControl>
+      <Header>
+        <Typography color={"primary"} bold={true}>
+          Manage Connections
+        </Typography>
+        <Button variant="outlined" value={NEW_SERVER_ID} key={NEW_SERVER_ID} onClick={() => onEditItem(emptyServer())}>
+          <Icon name="cloudDownload" />
+          New server
+        </Button>
+      </Header>
+      <EdsProvider density={theme}>
+        <Table style={{ width: "100%" }} className="serversList">
+          <Table.Head>
+            <Table.Row>
+              <Table.Cell style={CellHeaderStyle}>Server Name</Table.Cell>
+              <Table.Cell style={CellHeaderStyle}>Server</Table.Cell>
+              <Table.Cell style={CellHeader}>Test Connection</Table.Cell>
+              <Table.Cell style={CellHeaderStyle}>Status</Table.Cell>
+              <Table.Cell></Table.Cell>
+              <Table.Cell></Table.Cell>
+            </Table.Row>
+          </Table.Head>
+          <Table.Body>
+            {servers
+              .sort((a, b) => a.name.localeCompare(b.name))
+              .map((server: Server) => (
+                <Table.Row id={server.id} key={server.id}>
+                  <Table.Cell style={CellHeaderStyle}>{server.name}</Table.Cell>
+                  <Table.Cell style={CellHeaderStyle}>{server.url}</Table.Cell>
+                  <Table.Cell style={{ textAlign: "center" }}>
+                    <Icon color={selectedServer?.id == server.id ? colors.interactive.successResting : colors.text.staticIconsTertiary} name="cloudDownload" />
+                  </Table.Cell>
+                  <Table.Cell style={CellHeaderStyle}>
+                    <EdsProvider density={"compact"}>
```

Comment:
EDSProvider should not be used to style individual components according to the [documentation](https://eds-storybook-react.azurewebsites.net/?path=/docs/edsprovider--introduction).

Reply:
updated!